File: runtime-testsuite/test/org/antlr/v5/test/runtime/java/helpers/RuntimeTestParser.java
package org.antlr.v5.test.runtime.java.helpers;

import org.antlr.v5.runtime.core.Parser;
import org.antlr.v5.runtime.core.TokenStream;

public abstract class RuntimeTestParser extends Parser {
	protected java.io.PrintStream outStream = System.out;

	public RuntimeTestParser(TokenStream input) {
		super(input);
	}

	public void setOutStream(java.io.PrintStream outStream) {
		this.outStream = outStream;
	}
}

File: runtime-testsuite/test/org/antlr/v5/test/runtime/kotlin/helpers/RuntimeTestParser.kt
package org.antlr.v5.test.runtime.kotlin.helpers

import org.antlr.v5.runtime.core.TokenStream
import org.antlr.v5.runtime.core.Parser

abstract class RuntimeTestParser(input: TokenStream) : Parser(input) {
    var outStream: RuntimeTestPrintStream = RuntimeTestPrintStream(System.out)
}
